_data/simple-icons.json

```
@@ -8336,6 +8336,11 @@
             "source": "https://www.patreon.com/brand/downloads",
             "guidelines": "https://www.patreon.com/brand/downloads"
         },
+        {
+            "title": "Payhip",
+            "hex": "3197D6",
```

From the website's stylesheet, I think maybe the colour should be `#5c6ac4`
```suggestion
            "hex": "5C6AC4",
```